Source/Scenes/Details/XDetailsNode.swift

```diff
@@ -338,15 +345,15 @@ class XDetailsNode: ASCellNode {
         
```

Comment:
One way to increase the button height is
```
buttonsSpec.style.minHeight = ASDimensionMake(44)
```

Though I tried this and it resulted in a very tall button...that isn't right yet but it's better than making the text unnaturally big inside this button.

Reply:
The way I usually fix this is something like:

 ` button.contentEdgeInsets = UIEdgeInsets(top: 12, left: 0, bottom: 12, right: 0)`

On a real UIButton that extends the tap target of the button up and down from the visible area.  Facebook’s ASButtonNode acts similar to a UIButton, and when I saw it implements contentEdgeInsets I thought this would work.  But it makes the visible area larger too, and it does look too large for the view’s current design.  It’s funny, early in iOS days buttons looked just like that.  iOS 7 changed all that, and now roundrect buttons of that height look very out of place.